common/common-ui/src/main/java/com/duckduckgo/common/ui/view/shape/DaxBubbleCardViewExperiment.kt

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.common.ui.view.shape
+
+import android.content.Context
+import android.content.res.ColorStateList
+import android.util.AttributeSet
+import com.duckduckgo.common.ui.view.getColorFromAttr
+import com.duckduckgo.common.ui.view.toPx
+import com.duckduckgo.mobile.android.R
+import com.google.android.material.card.MaterialCardView
+import com.google.android.material.shape.EdgeTreatment
+import com.google.android.material.shape.ShapeAppearanceModel
+
+class DaxBubbleCardViewExperiment
```

Comment:
if this is an experiment, does it need to be in common-ui? I’d suggest to keep it where it’s used (app module) and only move it to common-ui once it’s approved.